src/Grpc.Net.Client/Internal/GrpcProtocolConstants.cs

```diff
@@ -103,15 +115,29 @@ static GrpcProtocolConstants()
             // but in case it isn't then don't include version in user-agent.
             if (assemblyVersion != null)
             {
-                userAgent += "/" + assemblyVersion.Version;
+                // 1.24.0
+                userAgent += $"/{assemblyVersion.Version} ";
             }
 
-            UserAgentHeader = "User-Agent";
-            UserAgentHeaderValue = userAgent;
-            TEHeader = "TE";
-            TEHeaderValue = "trailers";
+            // (.NET 5.0.7;
+            userAgent += $"({RuntimeInformation.FrameworkDescription}; ";
+            // CLR 5.0.0;
+            userAgent += $"CLR {Environment.Version}; ";
+
+            // .NETCoreApp,Version=v6.0; 
+            var targetFramework = assembly
```

Comment:
Grpc.Core currently produces strings like "net45", "netstandard2.0" etc.  Can we stay consistent? it's not an absolute must but there's value in consistency.

Reply:
Done! Let me know if you have any feedback on the formatting for this.